app/controllers/api/v1/users/omniauth_callbacks_controller.rb

```diff
@@ -4,17 +4,13 @@ module Api
   module V1
     module Users
       class OmniauthCallbacksController < Devise::OmniauthCallbacksController
+        rescue_from Exception, with: :render_error
```

Comment:
WHY U NO LIKE adding line breaks 😆 

```suggestion
        rescue_from Exception, with: :render_error
        
```

Reply:
🙈